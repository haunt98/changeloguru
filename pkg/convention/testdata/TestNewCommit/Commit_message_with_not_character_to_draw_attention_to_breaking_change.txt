refactor!: drop support for Node 6
