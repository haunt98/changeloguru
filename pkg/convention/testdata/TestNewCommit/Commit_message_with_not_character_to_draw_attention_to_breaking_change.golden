{
  "RawHeader": "refactor!: drop support for Node 6 0001-01-01",
  "Type": "refactor",
  "Scope": ""
}